From 24443b76f4daf74c21307044e81b4942bd4e586a Mon Sep 17 00:00:00 2001
From: Maciej Klemarczyk <m.klemarczyk+git@live.com>
Date: Sun, 28 Jun 2015 12:05:05 +0200
Subject: [PATCH 4/4] Missing LICENSE file was created

File was created based on license provided by README file.
---
 LICENCE | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)
 create mode 100644 LICENSE

diff --git LICENSE LICENSE
new file mode 100644
index 0000000..bdf271e
--- /dev/null
+++ LICENSE
@@ -0,0 +1,20 @@
+
+Copyright (C) 1999-2006 Takeshi Kanno
+Copyright (C) 2007-2009 Antony Dovgal
+
+This software is provided 'as-is', without any express or implied warranty.
+
+In no event will the authors be held liable for any damages arising from the 
+use of this software.
+
+Permission is granted to anyone to use this software for any purpose,including 
+commercial applications, and to alter it and redistribute it freely, subject 
+to the following restrictions:
+
+ 1. The origin of this software must not be misrepresented; you must not claim 
+    that you wrote the original software. If you use this software in a 
+    product, an acknowledgment in the product documentation would be 
+    appreciated but is not required.
+ 2. Altered source versions must be plainly marked as such, and must not be 
+    misrepresented as being the original software.
+ 3. This notice may not be removed or altered from any source distribution.
-- 
2.24.0

